
Well - some people have strange release policies - placing to be replaced
files online ... :-(

  - Rene Rebe <rene@exactcode.de>

diff -urN pl231src/prefabs/processdata.pl pl231src-patched/prefabs/processdata.pl
--- pl231src/prefabs/processdata.pl	2003-10-22 20:09:23.000000000 +0200
+++ pl231src-patched/prefabs/processdata.pl	2005-03-08 19:31:39.000000000 +0100
@@ -1,6 +1,7 @@
 #musthave action 
 
 #setifnotgiven data = "-"
+#setifnotgiven inlinedata = ""
 
 #include $chunk_read
 
diff -urN pl231src/src/proc_processdata.c pl231src-patched/src/proc_processdata.c
--- pl231src/src/proc_processdata.c	2005-01-26 17:47:26.000000000 +0100
+++ pl231src-patched/src/proc_processdata.c	2005-03-08 19:31:52.000000000 +0100
@@ -602,9 +602,9 @@
 		if( n > 1 ) setfloatvar( "SD", sqrt( ( accum[1] - (accum[0]*accum[0]/(double)n )) / ((double)n-1.0) ) );
 		else setcharvar( "SD", "n/a" ); 
 		setfloatvar( "MAX", max );
-		if( nfld > 1 ) setcharvar( "MAX_ID", breakbuf[0] );
+		if( tagfld >= 0 ) setcharvar( "MAX_ID", breakbuf[0] );
 		setfloatvar( "MIN", min );
-		if( nfld > 1 ) setcharvar( "MIN_ID", breakbuf[1] );
+		if( tagfld >= 0 ) setcharvar( "MIN_ID", breakbuf[1] );
 		}
 	return( 0 );
 	}
